packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -509,6 +504,73 @@ class PlatformZoomRange {
   final double? max;
 }
 
+class PlatformBitmap {
```

Comment:
This should get a comment explaining it, since the pattern here is non-obvious (due to the Pigeon limitation).